vm/hugepage/libhugetlbfs/dependency.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# For glibc-static dependence issue on rhel8
+# Bug 1567712 - glibc-static needs to be multilib on RHEL 8
+
+this_arch=$(uname -m)
+this_distro=$(grep DISTRO /etc/motd | cut -d '-' -f 3)
+pre_distro=${this_distro%.*}
+end_distro=${this_distro##*.}
+end_distro=${end_distro// /}
+
+if grep -q "release 8.*" /etc/redhat-release; then
+        if echo $this_distro | grep -q '.n'; then
+		choose=nightly
+        else
+		choose=rel-eng
+        fi
+        min_rel=$(grep -oE "[0-9]\.[0-9]" /etc/redhat-release | cut -d. -f2)
+
+	for ((; end_distro >= 0; end_distro-- )) do
```

Comment:
Can you please fix the indentation so it's consistent? The `if` block and the `min_rel` assignment are, for example, farther right than the `for` block.